tests/RPCServer.test.ts

```diff
@@ -1184,56 +1074,57 @@ describe(`${RPCServer.name}`, () => {
-            _cancel: (reason?: any) => void,
-            _meta: Record<string, JSONValue> | undefined,
-            ctx: ContextTimed,
-          ): AsyncGenerator<JSONRPCResponseResult> {
-            resolveCtxP(ctx);
-            yield* input;
-          };
-        }
-        const rpcServer = new RPCServer({
-          logger,
-          timeoutTime: lowerTimeoutTime,
-          idGen,
-        });
-        await rpcServer.start({
-          manifest: {
-            testMethod: new TestMethod({}),
-          },
-        });
-        const [outputResult, outputStream] = rpcTestUtils.streamToArray();
-        const readWriteStream: RPCStream<Uint8Array, Uint8Array> = {
-          cancel: () => {},
-          readable: stream,
-          writable: outputStream,
-        };
-        rpcServer.handleStream(readWriteStream);
-        await outputResult;
-        const ctx = await ctxP;
-        expect(ctx.timer.delay).toBe(higherTimeoutTime);
-        ctx.timer.cancel();
-        await ctx.timer.catch(() => {});
-      },
-    );
+    // Get feedback on this.
+    //   test.prop({
+    //     messages: specificMessageArb,
+    //     timeouts: rpcTestUtils.timeoutsArb,
+    //   })(
+    //     'handlers should be able to override lower timeout of RPCServer',
+    //     async ({ messages, timeouts: [lowerTimeoutTime, higherTimeoutTime] }) => {
+    //       const stream = rpcTestUtils.messagesToReadableStream(messages);
+    //       const { p: ctxP, resolveP: resolveCtxP } = promise<ContextTimed>();
+    //       class TestMethod extends DuplexHandler {
+    //         public timeout = higherTimeoutTime;
+    //         public handle = async function* (
+    //           input: AsyncGenerator<JSONRPCRequestParams>,
+    //           _cancel: (reason?: any) => void,
+    //           _meta: Record<string, JSONValue> | undefined,
+    //           ctx: ContextTimed,
+    //         ): AsyncGenerator<JSONRPCResponseResult> {
+    //           resolveCtxP(ctx);
+    //           yield* input;
+    //         };
+    //       }
+    //       const rpcServer = new RPCServer({
+    //         logger,
+    //         timeoutTime: lowerTimeoutTime,
+    //         idGen,
+    //       });
+    //       await rpcServer.start({
+    //         manifest: {
+    //           testMethod: new TestMethod({}),
+    //         },
+    //       });
+    //       const [outputResult, outputStream] = rpcTestUtils.streamToArray();
+    //       const readWriteStream: RPCStream<Uint8Array, Uint8Array> = {
+    //         cancel: () => {},
+    //         readable: stream,
+    //         writable: outputStream,
+    //       };
+    //       rpcServer.handleStream(readWriteStream);
+    //       await outputResult;
+    //       const ctx = await ctxP;
+    //       expect(ctx.timer.delay).toBe(higherTimeoutTime);
+    //       ctx.timer.cancel();
+    //       await ctx.timer.catch(() => {});
+    //     },
+    //   );
```

Comment:
Same here, checking the lower/higher timeout which I'm not sure if it is necessary or not.

Reply:
Same as what I commented before, we need both the tests to be present as they test for different behaviour.